tsl/src/planner.c

```diff
@@ -315,12 +315,47 @@ tsl_create_distributed_insert_path(PlannerInfo *root, ModifyTablePath *mtpath, I
 					if (rte->rtekind == RTE_SUBQUERY)
 					{
 						distributed = false;
+#if PG15_GE
```

Comment:
Does this new way works on the older PG versions? If it does, let's make it the only way, to support less variants of the code.

Reply:
Right, made code common for all versions.